src/dctl-transaction-create.js

```diff
@@ -20,7 +21,16 @@ program
     try {
       payload = JSON.parse(payload);
     } catch (e) {
-      console.warn('Could not parse JSON for payload, sending raw data instead...');
+      try {
+        const payloadSplit = payload.split('.');
```

Comment:
Instead of making this the fallback, fetching from a file should probably be a separate option. That way you can't 'accidentally' post a file, and it's easier to explicitly indicate as a user what you're trying to do (post data from the command line or read from a file instead)